### PR TITLE
[Impeller] add doc on iOS flamegraph capture.

### PR DESCRIPTION
**Pull request by jonahwilliams**
I find flamegraphs pretty useful, but Xcode doesn't have one built in that I know of.

### CODE REVIEW

File: impeller/docs/ios_cpu_profile.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+![Alt text](assets/ios_profiling/ios_time_profiler_example.png)
+
+  Click the red dot to begin and record as much profile data as you want, then press stop to conclude the trace.
+
+3. Select the thread to investigate, in this case you want io.flutter.raster. IMPORTANT: also select the trace root.
+
+![Alt text](assets/ios_profiling/ios_profiler_select_thread.png)
+
+4. Either via keyboard short cut or edit menu, "Deep Copy" the trace.
```
Comment (gaaclarke) [inline impeller/docs/ios_cpu_profile.md:17]:
```suggestion
4. Copy the trace with the keyboard shortcut or the menu ("Edit" -> "Deep Copy").
```

Comment (gaaclarke) [reply impeller/docs/ios_cpu_profile.md:17]: If we mention the keyboard shortcut we should mention what it is.

**Review by gaaclarke — APPROVED** [review]
lgtm

File: impeller/docs/ios_cpu_profile.md (hunk 1 above)
Comment (jonahwilliams) [reply impeller/docs/ios_cpu_profile.md]: Done

**Pull request merged**